Watching for file changes with StatReloader
[29/Mar/2025 16:13:35] "GET / HTTP/1.0" 200 13354
[29/Mar/2025 16:14:02] "GET /admin/ HTTP/1.0" 302 0
[29/Mar/2025 16:14:07] "POST /admin/login/?next=/blog/admin/ HTTP/1.0" 302 0
[29/Mar/2025 16:14:08] "GET /admin/blog/blogpost/ HTTP/1.0" 200 11729
[29/Mar/2025 16:14:12] "POST /admin/blog/blogpost/ HTTP/1.0" 200 9807
[29/Mar/2025 16:14:14] "GET /admin/blog/blogpost/ HTTP/1.0" 200 9025
[29/Mar/2025 16:14:58] "GET / HTTP/1.0" 200 4964
[29/Mar/2025 16:15:12] "GET / HTTP/1.0" 200 4964
[29/Mar/2025 16:15:42] "GET / HTTP/1.0" 200 4964
[29/Mar/2025 16:16:05] "GET / HTTP/1.0" 200 4964
[29/Mar/2025 16:16:20] "GET /create-blog/ HTTP/1.0" 302 0
[29/Mar/2025 16:17:05] "POST /login/?next=/blog/create-blog/ HTTP/1.0" 200 3439
[29/Mar/2025 16:17:19] "POST /register/ HTTP/1.0" 200 4557
[29/Mar/2025 16:17:31] "GET / HTTP/1.0" 200 4964
[29/Mar/2025 16:17:41] "GET / HTTP/1.0" 200 4964
[29/Mar/2025 16:17:51] "GET /login/?next=/blog/create-blog/ HTTP/1.0" 200 3436
[29/Mar/2025 16:17:56] "GET /create-blog/ HTTP/1.0" 200 5053
[29/Mar/2025 16:18:36] "POST /category/create/ HTTP/1.0" 302 0
/home/x017/my-projects/nginx-service/simple-blog/blog/models.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
March 29, 2025 - 16:09:09
Django version 5.1.7, using settings 'core.settings'
Starting development server at http://0.0.0.0:8001/
Quit the server with CONTROL-C.

Watching for file changes with StatReloader
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/lib/python3.12/threading.py", line 1075, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.12/threading.py", line 1012, in run
    self._target(*self._args, **self._kwargs)
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/core/management/commands/runserver.py", line 126, in inner_run
    autoreload.raise_last_exception()
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/utils/autoreload.py", line 87, in raise_last_exception
    raise _exception[1]
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/core/management/__init__.py", line 394, in execute
    autoreload.check_errors(django.setup)()
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/apps/registry.py", line 116, in populate
    app_config.import_models()
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/apps/config.py", line 269, in import_models
    self.models_module = import_module(models_module_name)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/x017/my-projects/nginx-service/simple-blog/blog/models.py", line 9
    creator = models.ForeignKey(User,on_delete=DO_NOTHING
                               ^
SyntaxError: '(' was never closed
/home/x017/my-projects/nginx-service/simple-blog/blog/models.py changed, reloading.
Watching for file changes with StatReloader
/home/x017/my-projects/nginx-service/simple-blog/blog/models.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
March 29, 2025 - 16:20:34
Django version 5.1.7, using settings 'core.settings'
Starting development server at http://0.0.0.0:8001/
Quit the server with CONTROL-C.

Watching for file changes with StatReloader
[29/Mar/2025 16:45:54] "GET / HTTP/1.0" 200 4964
[29/Mar/2025 16:48:59] "GET / HTTP/1.0" 200 4964
[29/Mar/2025 16:49:07] "GET /login HTTP/1.0" 301 0
[29/Mar/2025 16:49:12] "POST /login/ HTTP/1.0" 302 0
Internal Server Error: /blog/create-blog/
Traceback (most recent call last):
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/db/backends/sqlite3/base.py", line 354, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such column: blog_category.creator_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/backends/django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/defaulttags.py", line 199, in render
    len_values = len(values)
                 ^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/db/models/query.py", line 382, in __len__
    self._fetch_all()
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/db/models/query.py", line 1928, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/db/models/query.py", line 91, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/db/models/sql/compiler.py", line 1574, in execute_sql
    cursor.execute(sql, params)
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/db/backends/sqlite3/base.py", line 354, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such column: blog_category.creator_id
[29/Mar/2025 16:49:13] "GET /create-blog/ HTTP/1.0" 500 157763
[29/Mar/2025 16:49:41] "GET /create-blog/ HTTP/1.0" 200 5001
[29/Mar/2025 16:49:55] "POST /category/create/ HTTP/1.0" 302 0
[29/Mar/2025 16:50:02] "GET /create-blog/ HTTP/1.0" 200 5129
[29/Mar/2025 16:50:08] "GET / HTTP/1.0" 200 6474
[29/Mar/2025 16:50:16] "POST /create-blog/ HTTP/1.0" 302 0
[29/Mar/2025 16:50:40] "GET / HTTP/1.0" 200 8192
[29/Mar/2025 16:50:44] "GET /category/create/ HTTP/1.0" 200 2001
[29/Mar/2025 16:50:54] "POST /category/create/ HTTP/1.0" 302 0
[29/Mar/2025 16:51:04] "POST /create-blog/ HTTP/1.0" 302 0
[29/Mar/2025 16:51:26] "GET /create-blog/ HTTP/1.0" 200 5257
[29/Mar/2025 16:52:56] "GET /create-blog/ HTTP/1.0" 200 5542
[29/Mar/2025 16:53:08] "GET /create-blog/ HTTP/1.0" 200 5542
Internal Server Error: /blog/create-blog/
Traceback (most recent call last):
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/defaulttags.py", line 862, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 489, in parse
    raise self.error(token, e)
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 487, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 605, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 706, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '% if request.user == category.creator %' from '% if request.user == category.creator %'
[29/Mar/2025 16:55:21] "GET /create-blog/ HTTP/1.0" 500 173278
Internal Server Error: /blog/create-blog/
Traceback (most recent call last):
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/defaulttags.py", line 862, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 489, in parse
    raise self.error(token, e)
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 487, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 605, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 706, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '% if request.user == category.creator %' from '% if request.user == category.creator %'
[29/Mar/2025 16:55:56] "GET /create-blog/ HTTP/1.0" 500 173298
[29/Mar/2025 16:56:21] "GET /create-blog/ HTTP/1.0" 200 5489
[29/Mar/2025 16:56:27] "POST /category/create/ HTTP/1.0" 302 0
[29/Mar/2025 16:56:56] "GET /create-blog/ HTTP/1.0" 200 5592
[29/Mar/2025 16:57:28] "GET /create-blog/ HTTP/1.0" 200 5703
[29/Mar/2025 16:58:52] "GET /create-blog/ HTTP/1.0" 200 5610
[29/Mar/2025 16:59:47] "GET /create-blog/ HTTP/1.0" 200 5721
[29/Mar/2025 17:00:28] "GET /category/create/ HTTP/1.0" 200 2001
[29/Mar/2025 17:00:33] "GET /create-blog/ HTTP/1.0" 200 5864
/home/x017/my-projects/nginx-service/simple-blog/blog/views.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
March 29, 2025 - 16:21:11
Django version 5.1.7, using settings 'core.settings'
Starting development server at http://0.0.0.0:8001/
Quit the server with CONTROL-C.

Watching for file changes with StatReloader
[29/Mar/2025 17:01:50] "GET /category/create/ HTTP/1.0" 200 2001
[29/Mar/2025 17:01:53] "GET /create-blog/ HTTP/1.0" 200 6018
[29/Mar/2025 17:03:01] "GET /create-blog/ HTTP/1.0" 200 5928
[29/Mar/2025 17:04:04] "GET /admin/ HTTP/1.0" 200 9920
[29/Mar/2025 17:04:06] "GET /admin/jsi18n/ HTTP/1.0" 200 12738
[29/Mar/2025 17:04:13] "POST /admin/blog/category/ HTTP/1.0" 302 0
[29/Mar/2025 17:04:13] "GET /admin/jsi18n/ HTTP/1.0" 200 12738
[29/Mar/2025 17:04:57] "GET /create-blog/ HTTP/1.0" 200 4344
[29/Mar/2025 17:05:02] "POST /category/create/ HTTP/1.0" 302 0
[29/Mar/2025 17:05:20] "GET /create-blog/ HTTP/1.0" 200 4406
[29/Mar/2025 17:05:42] "GET /login/ HTTP/1.0" 200 3195
[29/Mar/2025 17:05:48] "GET /register/ HTTP/1.0" 200 3695
[29/Mar/2025 17:06:04] "GET / HTTP/1.0" 200 4964
[29/Mar/2025 17:06:11] "GET /create-blog/ HTTP/1.0" 200 4407
[29/Mar/2025 17:06:24] "POST /login/ HTTP/1.0" 302 0
[29/Mar/2025 17:06:26] "GET /create-blog/ HTTP/1.0" 200 4388
[29/Mar/2025 17:06:36] "GET / HTTP/1.0" 200 6492
Internal Server Error: /blog/create-blog/
Traceback (most recent call last):
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/defaulttags.py", line 862, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/defaulttags.py", line 961, in do_if
    condition = TemplateIfParser(parser, bits).parse()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/smartif.py", line 195, in parse
    retval = self.expression()
             ^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/smartif.py", line 206, in expression
    left = t.nud(self)
           ^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/smartif.py", line 149, in nud
    raise parser.error_class("Unexpected end of expression in if tag.")
django.template.exceptions.TemplateSyntaxError: Unexpected end of expression in if tag.
[29/Mar/2025 17:07:47] "GET /create-blog/ HTTP/1.0" 500 178271
[29/Mar/2025 17:08:44] "GET /create-blog/ HTTP/1.0" 200 4405
Internal Server Error: /blog/create-blog/
Traceback (most recent call last):
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/defaulttags.py", line 1531, in do_with
    extra_context = token_kwargs(remaining_bits, parser, support_legacy=True)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 1119, in token_kwargs
    kwargs[key] = parser.compile_filter(value)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 605, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 701, in __init__
    filter_func = parser.find_filter(filter_name)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 611, in find_filter
    raise TemplateSyntaxError("Invalid filter: '%s'" % filter_name)
django.template.exceptions.TemplateSyntaxError: Invalid filter: 'filter_creator'
[29/Mar/2025 17:31:36] "GET /create-blog/ HTTP/1.0" 500 169155
Internal Server Error: /blog/create-blog/
Traceback (most recent call last):
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/defaulttags.py", line 1531, in do_with
    extra_context = token_kwargs(remaining_bits, parser, support_legacy=True)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 1119, in token_kwargs
    kwargs[key] = parser.compile_filter(value)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 605, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 701, in __init__
    filter_func = parser.find_filter(filter_name)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/simple-blog/venv/lib/python3.12/site-packages/django/template/base.py", line 611, in find_filter
    raise TemplateSyntaxError("Invalid filter: '%s'" % filter_name)
django.template.exceptions.TemplateSyntaxError: Invalid filter: 'filter_creator'
[29/Mar/2025 17:31:36] "GET /create-blog/ HTTP/1.0" 500 169155
[29/Mar/2025 17:34:41] "GET / HTTP/1.0" 200 4354
[29/Mar/2025 17:34:47] "POST /category/create/ HTTP/1.0" 302 0
[29/Mar/2025 17:36:44] "GET /create-blog/ HTTP/1.0" 200 4847
[29/Mar/2025 17:36:49] "POST /category/create/ HTTP/1.0" 302 0
[29/Mar/2025 17:36:58] "GET /login/ HTTP/1.0" 200 3428
[29/Mar/2025 17:37:07] "GET / HTTP/1.0" 200 4353
[29/Mar/2025 17:37:23] "POST /category/create/ HTTP/1.0" 302 0
[29/Mar/2025 17:37:30] "POST /create-blog/ HTTP/1.0" 302 0
[29/Mar/2025 17:38:39] "GET / HTTP/1.0" 200 5373
